docs/content/modules/tensor_mechanics/systems/Kernels/StressDivergenceRSphericalTensors.md

```diff
@@ -1,8 +1,24 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#StressDivergenceRSphericalTensors
+#Stress Divergence RSpherical Tensors
+!description /Kernels/StressDivergenceRSphericalTensors
+
 !devel /Kernels/StressDivergenceRSphericalTensors float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
-!description /Kernels/StressDivergenceRSphericalTensors
+##Description
+This kernel solves the steady state stress divergence equation in polar coordinates on a 1D mesh.
+
+{!content/modules/tensor_mechanics/CommonDocumentation/StressDivergenceKernels.md!}
```

Comment:
This ability to link to a separate file to duplicate content is quite handy.  I made a separate folder for these *.md files that are common to multiple material/kernel descriptions at the same level as `tensor_mechanics/systems` but did not include the folder in the `pages.yml` file.  Is this separate folder okay?  My goal was to avoid completely overfilling the Materials folder